common/config/rush/browser-approved-packages.json

```diff
@@ -1,4 +1,9 @@
 // DO NOT ADD COMMENTS IN THIS FILE.  They will be lost when the Rush tool resaves it.
 {
-  "packages": []
+  "packages": [
+    {
+      "name": "strict-uri-encode",
```

Comment:
>strict [](start = 15, length = 6)

Approved because:

1. This is MIT license
2. It has no other dependencies
3. It's what PNPM uses